### PR TITLE
Filling links in section 3

### PR DESCRIPTION
**Pull request by jcommelin**

### CODE REVIEW

File: data/overview.yaml [hunk 1]
```diff
@@ -115,69 +115,69 @@ Groups:
...
+      Characteristic p: 'algebra/char_p.html#char_p'
   subfields: 'field_theory/subfield.html'
   Frobenius morphisms: 'algebra/char_p.html#frobenius' 
   field extensions:
   integral ring fractional fields:
   Field Q of rational numbers: 'data/rat/basic.html#rat.division_ring'
   Field R of real numbers: 'data/real/basic.html#real.division_ring'
   Field C of complex numbers: 'data/complex/basic.html#complex.field
-  D'Alembert-Gauss theorem: 'analysis/complex/polynomial.html#complex.exists_root'
+  "D'Alembert-Gauss theorem": 'analysis/complex/polynomial.html#complex.exists_root'
```
Comment (RaitoBezarius) [inline data/overview.yaml:175]: This is not the issue, the above thing is not terminated by a `'`, this was my bad, I fixed it in my PR, if you add it, Git is smart enough to not create conflicts, so please go ahead.

File: data/overview.yaml [hunk 2]
```diff
@@ -115,69 +115,69 @@ Groups:
...
   Unique factorisation domain (UFD): 'ring_theory/unique_factorization_domain.html'
-  Greatest common divisor:
-  Least common multiple:
-  A[X] is a UFD when A is a UFD:
+  Greatest common divisor: 'algebra/gcd_domain.html#gcd_domain.gcd'
+  Least common multiple: 'algebra/gcd_domain.html#gcd_domain.lcm'
+  A[X] is a UFD when A is a UFD: ''
   Principal ring: 'ring_theory/principal_ideal_domain.html#submodule.is_principal'
-  Bezouts theorem:
+  Bezout's theorem: ''
```
Comment (PatrickMassot) [inline data/overview.yaml:141]: Are you sure we don't have this?

File: data/overview.yaml [hunk 3]
```diff
@@ -115,69 +115,69 @@ Groups:
...
-      Characteristic p: 'algebra/char_zero.html#char_zero'
+      Characteristic zero: 'algebra/char_zero.html#char_zero'
+      Characteristic p: 'algebra/char_p.html#char_p'
   subfields: 'field_theory/subfield.html'
   Frobenius morphisms: 'algebra/char_p.html#frobenius' 
   field extensions:
   integral ring fractional fields:
   Field Q of rational numbers: 'data/rat/basic.html#rat.division_ring'
   Field R of real numbers: 'data/real/basic.html#real.division_ring'
   Field C of complex numbers: 'data/complex/basic.html#complex.field
```
Comment (jcommelin) [inline data/overview.yaml:174]:
```suggestion
  Field C of complex numbers: 'data/complex/basic.html#complex.field'
```

File: data/overview.yaml [hunk 4]
```diff
@@ -115,69 +115,69 @@ Groups:
...
-  D'Alembert-Gauss theorem: 'analysis/complex/polynomial.html#complex.exists_root'
+  "D'Alembert-Gauss theorem": 'analysis/complex/polynomial.html#complex.exists_root'
   Algebraic elements:
   Transcendental elements:
   Algebraic extensions:
   Algebraically closed fields:
-  Rupture fields:
+  Rupture fields: 'ring_theory/adjoin_root.html#adjoin_root'
   Splitting fields:
```
Comment (PatrickMassot) [inline data/overview.yaml:181]: Don't we have that?

File: data/overview.yaml [hunk 5]
```diff
@@ -115,69 +115,69 @@ Groups:
...
   Finite fields: 'field_theory/finite.html'
   Rational fraction fields with one indeterminate over the field:
```
Comment (PatrickMassot) [inline data/overview.yaml:183]: Isn't this offered by fraction fields? I guess it's fair to wait until we have some convenient API anyway.

File: data/overview.yaml (hunk 4 above)
Comment (RaitoBezarius) [reply data/overview.yaml:181]: According to Kevin, no, see the discussion in Zulip

**Pull request merged**